bynary/apps/demo/src/app/core/feature/recipe/pages/detail/recipe-detail.component.spec.ts

```diff
@@ -138,43 +140,80 @@ describe('RecipeDetailComponent', () => {
                 expect(recipeDescription.nativeElement.textContent).toEqual(' Bla bla ');
             });
 
-            it('should have a ingredients', () => {
+            it('should have ingredients', () => {
                 const recipeIngredients = debugElement.queryAll(By.css('.c-recipe-detail-component__recipe-ingredients'));
 
                 expect(recipeIngredients).toHaveLength(2);
-                expect(recipeIngredients[0].nativeElement.textContent).toEqual(' Schnitzel - 2 ');
-                expect(recipeIngredients[1].nativeElement.textContent).toEqual(' Pommes - 1 ');
             });
```

Comment:
Den Part kannst du eigtl auch noch in den Describe-Block `ingredients` schreiben, dann müsstest du nicht doppelt die `recipeIngredients` querien.

`describe('ingredients' ...)`

`beforeEach(() => { recipeIngredients = ... })`

`it('should have 2 ingredients, () => ...)`